concepts/profile/realworldusage.md

```diff
@@ -0,0 +1,135 @@
+# Real World Usage
+
+This page backs away from our profile-focused data view and examines what Firefox Desktop usage looks like in the real world. There are many components and layers that exist between a user acquiring and running Firefox, and this documentation will illuminate what those are and how they can affect the meaning of a profile. 
+
+## Real Life Components of Firefox Desktop Usage
+
+![](images/real-life-usage-components.png)
+
+The above image illustrates all the layers that sit between a user acquiring and running Firefox Desktop and the Telemetry pings we receive. 
+
+* 1: The user
+	- A human being presumably. 
+* 2: The machine
+	- The physical hardware running Firefox. 
+* 3: The disk image / hard drive
+	- A single machine could have separate partitions running different OS. 
+	- Or conversely, they could copy the disk image and run it on multiple machines. 
+	- Disk images are also used as backups to restore a machine. 
+* 4: OS user profile
+	- Most operating systems allow users to log into different user profiles with separate user directories (such as a "Guest" accounts). 
+	- Usually, Firefox is installed into a system directory that all users profiles will share, but Firefox profiles are saved within the user directories, effectively segregating them. 
+* 5: Firefox binary / installer
+	- The downloaded binary package or stub installer which installs Firefox into the hard drive. Users can get these from our website or one of our managed properties, but they can also acquire these from 3rd party sources as well. 
+	- Our website is instrumented with Google Analytics to track download numbers, but other properties (FTP) and 3rd party sources are not. Google Analytics data is not directly connected to Telemetry data. 
```

Comment:
At this point it might be relevant to mention attribution's role in identifying the source of the installed firefox.

Reply:
good point. i'm also working on separate documentation explaining the attribution process, will link it to this doc in the future.